docs/docs/dev-manuals/services/invoice.mdx

```diff
@@ -0,0 +1,126 @@
+import CustomImage from "@site/src/components/CustomImage";
+
+# Invoice Service
+
+The invoice service provides the bounded context `Invoice`.
+It is responsible for managing invoices of orders.
+
+## Domain Model
+
+// TODO: Niklas
+
+<CustomImage path="/renderedDiagrams/invoiceDomainModel" width="1365" height="131" />
+
+:::info
+
+Imported entities from other bounded contexts are marked with a dashed border.
+
+:::
+
+## API
+
+The service provides a GraphQL API, which is documented in the [GraphQL API documentation](/docs/graphql/invoice).
+
+## Technology Stack
+
+- **Language**: Rust
+- **Framework**: [axum](https://docs.rs/axum/latest/axum/)
+- **GraphQL Library**: [async_graphql](https://docs.rs/async-graphql/latest/async_graphql/)
+- **Database**: MongoDB
+
+## Repository Structure
+
+The repository is structured as follows:
+
+<div className="repository-structure">
+ - `/src`: Source code of the service - `authentication.rs` Header authentication logic of service -
```

Comment:
The problematic structure is present in all new files apparently.
Please convert it to (nested) lists again.

Reply:
This was caused by Prettier, if someone has a working instance feel free to format :)